assets/css/styles.scss

```diff
@@ -17,6 +17,9 @@ $breakpoints: (
   xxl: 1408px,
 );
 
+// Maximum width of site contents
+$container-max-width: 1280px;
```

Comment:
is this true going forward for **all** containers?

Reply:
This would be the way going forward and is currently applicable for the announcements and text cards components.

Reply:
My bad, this is mostly true unless the component gets squished. From design:

> Infocol will use 1280px! The Text Cards use 1440px b/c the 4-cards layout gets very squished at 1280px